package.json

```diff
@@ -1,11 +1,15 @@
 {
-  "name": "@salesforce/salesforce-sdk",
+  "name": "sf-fx-sdk-nodejs",
```

Comment:
I'm confused why this isn't namespaced under `@heroku/*`? :-)

Reply:
or `@salesforce` for that matter. :)